commands/team.go

```diff
@@ -222,6 +233,43 @@ func deleteTeam(c client.Client, team *model.Team) (bool, *model.Response) {
 	return c.PermanentDeleteTeam(team.Id)
 }
 
+func archiveTeamsCmdF(c client.Client, cmd *cobra.Command, args []string) error {
+	if len(args) < 1 {
```

Comment:
You can replace this check by adding `Args: cobra.MinimumNArgs(1),` to the command definition

Reply:
Done.